apps/webapp/app/routes/resources.orgs.$organizationSlug.select-plan.tsx

```diff
@@ -242,16 +373,11 @@ export function TierFree({
           </div>
         </div>
       ) : (
```

Comment:
This change moves the Form inside the Digalog, rather than wrapping it. The Form wasn't behaving well when the action to submit the form was inside the Dialog. This changed required adding 2 Dialogs (1 for Github verification and 1 for the downgrade) so i had to duplicate the Form to add to each Dialog and used a unique id to make sure they submitted correctly.